src/components/Banner/index.tsx

```diff
@@ -117,6 +121,21 @@ function Banner({
   const theme = useTheme()
 
   const ALL_BANNERS = [
+    {
+      // KyberSwap Trading Campaign Cronos
+      id: 'kyberSwap-trading-campaign-cronos',
+      start: new Date(1658793600000), // July 26, 2022 0:00:00
```

Comment:
You can write like this `new Date('Jul 26 2022 00:00:00 GMT+0700')`

Reply:
Or `new Date(2022, 6, 26)`

Reply:
@hungdoansy cool

Reply:
this's better `new Date('2022-07-22T00:00:00.000Z')`, use for gmt

Reply:
use epoch for easier read for now

Reply:
+1 for  @namgold

Reply:
Note for next time :tien-sword:

Reply:
when next time...

Reply:
or `Date.UTC(year, month, day, hour, minute, second, millisecond)`